pages/treeherder.py

```diff
@@ -13,11 +13,12 @@
 
 class TreeherderPage(Base):
 
+    _first_resultset_datestamp_locator = (By.CSS_SELECTOR, '.result-set .result-set-title-left > span a')
     _job_details_actionbar_locator = (By.ID, 'job-details-actionbar')
     _job_result_status_locator = (By.CSS_SELECTOR, '#result-status-pane > div:nth-child(1) > span')
     _logviewer_button_locator = (By.ID, 'logviewer-btn')
-    _resultset_locator = (By.CSS_SELECTOR, 'div.row.result-set')
-    _result_status_locator = (By.ID, 'job-details-panel')
+    _results_locator = (By.CSS_SELECTOR, '.result-set-bar')
+    _single_resultset_datestamp_locator = (By.CSS_SELECTOR, '.result-set-title-left > span:nth-child(2) > a:nth-child(1)')
```

Comment:
When there's only one resultset, our `_first_resultset_datestamp_locator` should still work.